### DISCUSSION TITLE
v0.10.6

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#841] [Announcements]
https://github.com/Blinue/Magpie/compare/v0.10.5...v0.10.6

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.10.6'>v0.10.6</a>.</em>

**Comment by knightriot** [comment]
Hey bro, can i have your contact (email or telegram ... etc)
